### PR TITLE
server: Add TTL for ephemeral container db entries

### PR DESCRIPTION
**Pull request by znewton**
## Description

We delete Ephemeral Containers after session close (10 min after NoClient Op in Deli). For summaries, we set a 24 hour TTL in Redis. We should also add a 24 hour TTL in the DB.

### CODE REVIEW

File: server/routerlicious/packages/services-shared/src/storage.ts
Comment (zhangxin511) [inline server/routerlicious/packages/services-shared/src/storage.ts:292]: We can define `60*60*24` as const first

Comment (zhangxin511) [inline server/routerlicious/packages/services-shared/src/storage.ts:292]: Two things:
1. Let's get @yangg-msft  and @pradeepvairamani  feedback on this
2. Once merged, we need to set a colllection ttl that never expire, and this document level would override that behavior to have functional

**Review by pradeepvairamani — APPROVED** [review]

**Comment by znewton** [comment]
Went a different route with this

File: server/routerlicious/packages/services-shared/src/storage.ts
Comment (znewton) [reply server/routerlicious/packages/services-shared/src/storage.ts:293]: I think this is OK. in other places, we use an indecipherable number, which is the result of this multiplication, so I feel this is already more readable than other spots. Maybe worth a follow-up up though.

Comment (znewton) [reply server/routerlicious/packages/services-shared/src/storage.ts:293]: I think this can be merged without risk, and once the TTL is added on FRS side, it will allow cleanup to happen automatically.

**Pull request merged**
